conn.go

```diff
@@ -15,15 +15,22 @@
 package nftables
 
 import (
+	"errors"
 	"fmt"
 	"sync"
 
+	"github.com/google/nftables/binaryutil"
 	"github.com/google/nftables/expr"
 	"github.com/mdlayher/netlink"
 	"github.com/mdlayher/netlink/nltest"
 	"golang.org/x/sys/unix"
 )
 
+// errMsgNoError is returned in cases when netlink returns an error
+// message with a status 0, which means that there is no error
+// https://github.com/mdlayher/netlink/blob/main/message.go#L285-L289
```

Comment:
Use the `y` keyboard shortcut on GitHub to turn this into a more stable URL: https://github.com/mdlayher/netlink/blob/9a593f9dc1a92ae1c7c45a9fc663b7dd1e111eac/message.go#L285-L289 :)